app/controllers/courses_controller.rb

```diff
@@ -385,6 +394,7 @@ def upload_roster
       begin
         save_uploaded_roster
         flash.now[:success] = "Successfully updated roster!"
```

Comment:
Now that you're redirecting, this should be `flash[:success] = ...`

In general, `flash.now` is only when we are directly rendering a page, rather than redirecting.

Reply:
Good call, changed